.github/workflows/ci.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     runs-on: ubuntu-latest
     container: integratedexperts/ci-builder
     env:
-      CKAN_REPO: qld-gov-au/ckan
+      CKAN_REPO: ckan/ckan
```

Comment:
can you put the ckan_repo into the matrix so we can test against ckan/ckan and qld-gov-au/ckan ??

Reply:
I could, if you think it's beneficial. It would mean a lot more GitHub Actions runs, and there shouldn't be any relevant difference between the two, but, I suppose if we did something badly wrong it could come up.

Reply:
in retrospect, can we get our ckan tweaks into 2.9 codebase upstream so we can drop our own version.

Reply:
It's a good long term goal.

Reply:
Okay, https://github.com/frictionlessdata/ckanext-validation/pull/57 only has our latest `master` changes, which are a bit behind, but it's a start.